packages/babel-plugin-transform-computed-properties/test/fixtures/spec/spread-element-lose/output.js

```diff
@@ -0,0 +1,11 @@
+const first = {
+  b: 'b'
+};
+const third = {
+  d: 'd'
+};
+const second = babelHelpers.defineProperty({
+  ...first,
+  a: 'a',
+  ...third
+}, 'c', 'c');
```

Comment:
This is wrong: if `third` contains `c: 1`, `second` should contain `c: 1` but with this output code it will contain `c: 'c'`. You should take a look at how we make sure that normal properies that come after the computer property are defined after the `defineProperty` call, and do the same.